auth-web/package-lock.json

```diff
@@ -47,7 +47,7 @@
         "@types/vuelidate": "^0.7.13",
         "@typescript-eslint/eslint-plugin": "^5.23.0",
         "@typescript-eslint/parser": "^5.23.0",
-        "@volar-plugins/vetur": "*",
+        "@volar-plugins/vetur": "latest",
```

Comment:
Did you change this?

Reply:
I did not

Reply:
Weird. I just added the package to Filings UI (to see what it does) and it looks like this for me:
```
"@volar-plugins/vetur": "^0.1.0",
```

Reply:
@seeker25 What's this package about, and why did the spec change?

Reply:
I think its a problem with you npm config files.

Reply:
just ran 'npm i'. You think reverting back to the way it was good idea?

Reply:
Let's see what Travis knows about this. In the meantime, I'd be tempted to revert this line.

Reply:
I will wait for Travis's reply :)
I am on vacation starting tomorrow but will check this sometime tomorrow and will take it from there.

Reply:
I changed it, was recommendations straight out of VSCODE.

Reply:
Oh, OK. So Ketaki is missing a rebase.